test/SpokePool.SlowRelay.ts

```diff
@@ -221,6 +221,7 @@ describe("SpokePool Slow Relay Logic", async function () {
           erc20Message,
           0, // Should not have an applied relayerFeePct for slow relay fills.
           true,
+          "9000000000000000000",
```

Comment:
Same -- how was this working before?

Reply:
My understanding is that this was pre ACX 2.5. Here's a passing test from several weeks ago: https://github.com/across-protocol/contracts-v2/actions/runs/4835389290/jobs/8617643141